src/accelerate/accelerator.py

```diff
@@ -1383,6 +1416,15 @@ def _get_devices(self, *args):
         return (model_device, optimizer_device)
 
     def get_state_dict(self, model, unwrap=True):
+        """
+        Returns the state dictionary of a model sent through [`Accelerator.prepare`] in full precision
+
+        Args:
+            model (`Module`, `DistributedDataParallel`, `FullyShardedDataParallel`, `MpModelWrapper`):
```

Comment:
I think the other types are all subclasses of `Module`
```suggestion
            model (`torch.nn.Module`):
```